components/constellation/constellation.rs

```diff
@@ -2841,7 +2842,8 @@ where
             top_level_browsing_context_id,
             browsing_context_id,
             new_pipeline_id,
-            replace: None,
+            // pipeline already closed, so Yes avoids harmless but useless close
```

Comment:
Would it be possible to expand this comment a bit? Why is the pipeline already closed and where is the close avoided?

Reply:
done in c1a6558ca29daeb81020934a828e6cf0014f99f8